server/app.js

```diff
@@ -14,19 +15,23 @@ import {
 const apiRoot = "/api";
 const studentRoot = "/api/student";
 
```

Comment:
Hi , I think you missing a variable here

Reply:
still working on it  can you advise how to connect Postgres to server again